# WARNING:  This is a quick and dirty hack to get easily the dataset columns
#           It is much safer to retrieve this parameter from the pose_dataset_builder_node
dataset_columns: [user_id, time_stamp,
  head_pos_x, head_pos_y, head_pos_z, head_orient_x, head_orient_y,
  head_orient_z, head_orient_w, head_pos_confidence, head_orient_confidence, neck_pos_x,
  neck_pos_y, neck_pos_z, neck_orient_x, neck_orient_y, neck_orient_z, neck_orient_w,
  neck_pos_confidence, neck_orient_confidence, torso_pos_x, torso_pos_y, torso_pos_z,
  torso_orient_x, torso_orient_y, torso_orient_z, torso_orient_w, torso_pos_confidence,
  torso_orient_confidence, left_shoulder_pos_x, left_shoulder_pos_y, left_shoulder_pos_z,
  left_shoulder_orient_x, left_shoulder_orient_y, left_shoulder_orient_z, left_shoulder_orient_w,
  left_shoulder_pos_confidence, left_shoulder_orient_confidence, left_elbow_pos_x,
  left_elbow_pos_y, left_elbow_pos_z, left_elbow_orient_x, left_elbow_orient_y, left_elbow_orient_z,
  left_elbow_orient_w, left_elbow_pos_confidence, left_elbow_orient_confidence, left_hand_pos_x,
  left_hand_pos_y, left_hand_pos_z, left_hand_orient_x, left_hand_orient_y, left_hand_orient_z,
  left_hand_orient_w, left_hand_pos_confidence, left_hand_orient_confidence, right_shoulder_pos_x,
  right_shoulder_pos_y, right_shoulder_pos_z, right_shoulder_orient_x, right_shoulder_orient_y,
  right_shoulder_orient_z, right_shoulder_orient_w, right_shoulder_pos_confidence,
  right_shoulder_orient_confidence, right_elbow_pos_x, right_elbow_pos_y, right_elbow_pos_z,
  right_elbow_orient_x, right_elbow_orient_y, right_elbow_orient_z, right_elbow_orient_w,
  right_elbow_pos_confidence, right_elbow_orient_confidence, right_hand_pos_x, right_hand_pos_y,
  right_hand_pos_z, right_hand_orient_x, right_hand_orient_y, right_hand_orient_z,
  right_hand_orient_w, right_hand_pos_confidence, right_hand_orient_confidence, left_hip_pos_x,
  left_hip_pos_y, left_hip_pos_z, left_hip_orient_x, left_hip_orient_y, left_hip_orient_z,
  left_hip_orient_w, left_hip_pos_confidence, left_hip_orient_confidence, left_knee_pos_x,
  left_knee_pos_y, left_knee_pos_z, left_knee_orient_x, left_knee_orient_y, left_knee_orient_z,
  left_knee_orient_w, left_knee_pos_confidence, left_knee_orient_confidence, left_foot_pos_x,
  left_foot_pos_y, left_foot_pos_z, left_foot_orient_x, left_foot_orient_y, left_foot_orient_z,
  left_foot_orient_w, left_foot_pos_confidence, left_foot_orient_confidence, right_hip_pos_x,
  right_hip_pos_y, right_hip_pos_z, right_hip_orient_x, right_hip_orient_y, right_hip_orient_z,
  right_hip_orient_w, right_hip_pos_confidence, right_hip_orient_confidence, right_knee_pos_x,
  right_knee_pos_y, right_knee_pos_z, right_knee_orient_x, right_knee_orient_y, right_knee_orient_z,
  right_knee_orient_w, right_knee_pos_confidence, right_knee_orient_confidence, right_foot_pos_x,
  right_foot_pos_y, right_foot_pos_z, right_foot_orient_x, right_foot_orient_y, right_foot_orient_z,
  right_foot_orient_w, right_foot_pos_confidence, right_foot_orient_confidence, pose]
